src/components/node-list/node-list-tree.js

```diff
@@ -194,6 +220,7 @@ const TreeListProvider = ({
       className={classes.root}
       defaultCollapseIcon={<ExpandMoreIcon />}
       defaultExpandIcon={<ChevronRightIcon />}
+      onNodeToggle={onItemExpandToggle}
```

Comment:
naming it as `onNodeToggle` could be quite confusing - this might be better to be renamed as `onPipelineToggle`?

Reply:
This is the name of the prop on material-ui `StyledTreeView` component so think we'll need to keep it?

Reply:
indeed - thanks for clarifying! Funny it does sound exactly like how we would name our callback prop :grin: